catalog.html

```diff
@@ -220,8 +256,52 @@ <h3>COMPANY</h3>
             </div>
             <div class="copyright">
                 <p>BINABOX <time>2022</time> -&copy;ALL rights reserved</p>
+            
+                <ul class="social-links-list">
+                    <li class="social-links-items-footer-colored">
+                        <a href="" class="social-link">
+                            <svg class="social-link-ico">
+                                <use href="/img/sprite.svg#xtwi" width="18"     height="16"></use>
+                            </svg>
```

Comment:
You used `width` and `height` attributes on `<use>` tag. It's valid in terms of HTML validity. This way you specify size of used symbol from the sprite. But it's wrong if you want to specify entire SVG icon size.

I would recommend you to move sizes to parent `<svg>` tag.

p.s. you're using a lot of extra spaces, it's also bad practice in terms of code style as in previous comment.

```suggestion
                            <svg class="social-link-ico" width="18" height="16">
                                <use href="/img/sprite.svg#xtwi"></use>
                            </svg>
```